### ISSUE TITLE
Привет от wemake-python-styleguide

### ISSUE THREAD

**Issue by sobolevn** [ShashkovS/flake8_ejudge#1]
@ShashkovS привет! Наткнулся на вашу библиотеку. Большое спасибо, что разрабатываете и поддерживаете ее.

Я написал довольно много правил для flake8 в рамках нашей компании: https://github.com/wemake-services/wemake-python-styleguide

Возможно, вам было бы интересно поставить какие-то (или все) из них для проверки работ в школе. Ученикам - очень полезно! Проверено на моих студентах 😉 

Если есть вопросы - буду рад ответить!

**Comment by ShashkovS** [comment]
@sobolevn Да, я ваш проект видел и даже у меня были планы его прикрутить.
Я планировал запустить проверку стилей на наших 200,000 посылок школьников, чтобы посмотреть, какие правила точно нужно отключить. Но какие-то проверки из вашего проекта мне прямо хочется добавить.
